webapp/locales/de.json

```
@@ -484,8 +485,9 @@
         },
         "teaserImage": {
             "cropperConfirm": "Bestätigen"
-        },
-        "languageSelectText": "Sprache wählen"
+        },        
+        "inappropriatePicture" : "Dieses Bild kann für einige Menschen unangemessen sein.",
+        "inappropriatePictureText" : "Beitragsbild wird Verschwommen dargestellt"
```

one last, tiny change – since this is the link text that leads to the FAQ, I would suggest something like:
```suggestion
        "inappropriatePictureText" : "Wann soll ein Foto versteckt werden"
```

---

ok